apps/condo/domains/acquiring/schema/Payment.js

```diff
@@ -92,27 +106,27 @@ const Payment = new GQLListSchema('Payment', {
 
         advancedAt: {
             schemaDoc: 'Time at which money was advanced to recipient\'s account',
-            type: DateTimeUtc,
+            type: 'DateTimeUtc',
             isRequired: false,
         },
 
         accountNumber: {
             schemaDoc: 'Payer\'s account number',
-            type: Text,
+            type: 'Text',
```

Comment:
We can add a trim to wrapper

Reply:
If we will need some processing, we will create a custom field type.